### PR TITLE
Add recipe for supersmoother

### PR DESCRIPTION
**Pull request by cbcunc**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by cbcunc** [comment]
@conda-forge/help-python Review requested

File: recipes/supersmoother/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - python
+  run:
+    - python
+    - numpy
+    - scipy
+
+test:
+  imports:
+    - supersmoother
+    - supersmoother.tests
```
Comment (mwcraig) [inline recipes/supersmoother/meta.yaml]: You can drop the import test for the tests

File: recipes/supersmoother/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "aeef4e1b00c32316d624ea7e3ac87c244bf2e59abbb6c042e7791f69ae0669cb"
+
+build:
+  number: 0
```
Comment (mwcraig) [inline recipes/supersmoother/meta.yaml]: I think you could make this a `noarch` package, which will mean less CI usage once it is merged. See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

**Comment by cbcunc** [comment]
@mwcraig Thanks for the help. I will make these changes either later tonight or first thing in the morning. Headed to the Clay Pit where I hope to see you.

**Comment by mwcraig** [comment]
Thanks again for the recipe!

**Comment by cbcunc** [comment]
@conda-forge/help-python Moar review requested

**Pull request merged**